### ISSUE TITLE
Modrinth App refuses to load after updated (Linux)

### ISSUE THREAD

**Issue by NoamRothschild** [modrinth/code#2363]
### Please confirm the following.

- [X] I checked the [existing issues](https://github.com/modrinth/code/issues) for duplicate problems
- [X] I have tried resolving the issue using the [support portal](https://support.modrinth.com)
- [X] I have ensured my Modrinth App installation is up to date

### What version of the Modrinth App are you using?

Unable to check (latest as of 6th September 2024)

### What operating systems are you seeing the problem on?

Linux

## Hello,
I have been using the modrinth app on my linux machine (Ubuntu 24.04), and a prompt asking me to update the software to the latest version was shown (nothing too special), I tried updating it & it did not look like it failed but when I tried opening the app again again it refused to load correctly as show bellow:

![image](https://github.com/user-attachments/assets/1af90a5c-01cd-415f-921f-a0466ffdff98)

Upon further inspection I was able to get some logs of the app by running it directly from the termial:
```bash
~/.AppImages 
❯ ./modrinth-app_0.7.1_amd64.AppImage 
/usr/lib/x86_64-linux-gnu/gvfs/libgvfscommon.so: undefined symbol: g_task_set_static_name
Failed to load module: /usr/lib/x86_64-linux-gnu/gio/modules/libgvfsdbus.so
/usr/lib/x86_64-linux-gnu/gvfs/libgvfscommon.so: undefined symbol: g_task_set_static_name
Failed to load module: /usr/lib/x86_64-linux-gnu/gio/modules/libgvfsdbus.so
/usr/lib/x86_64-linux-gnu/gvfs/libgvfscommon.so: undefined symbol: g_task_set_static_name
Failed to load module: /usr/lib/x86_64-linux-gnu/gio/modules/libgvfsdbus.so
KMS: DRM_IOCTL_MODE_CREATE_DUMB failed: Permission denied
Failed to create GBM buffer of size 1280x800: Permission denied
/lib/x86_64-linux-gnu/libcurl-gnutls.so.4: undefined symbol: nghttp2_option_set_no_rfc9113_leading_and_trailing_ws_validation
Failed to load module: /usr/lib/x86_64-linux-gnu/gio/modules/libgiolibproxy.so
/usr/lib/x86_64-linux-gnu/gio/modules/libdconfsettings.so: undefined symbol: g_assertion_message_cmpint
Failed to load module: /usr/lib/x86_64-linux-gnu/gio/modules/libdconfsettings.so

** (theseus_gui:16611): WARNING **: 11:04:01.808: atk-bridge: get_device_events_reply: unknown signature

~/.AppImages took 1m49s 
❯ 
```

Please also note that the "Permission denied" in the error was not fixed by running the app as **sudo** and changed nothing.

### Expected behavior

In the times when I rejected the update, the app launched successfully, this was the fist time I tried to update the app and it broke it.

## TL;DR
App does not work after updating on a Linux machine (tested on Ubuntu 24.04)

**Comment by skurpi** [comment]
Can confirm this is still a problem. I'm using Arch Linux, installing Modrinth through flatpak. I've gone back to a previous version for now.

**Comment by Maweypeyyu** [comment]
As a temporary workaround try 

```sh
env WEBKIT_DISABLE_COMPOSITING_MODE=1 ./modrinth.AppImage
```

**Comment by Prospector** [comment]
Consolidating into #3057
